### ISSUE TITLE
Form Designers

### ISSUE THREAD

**Issue by tataDan** [dontpanic92/wxGo#68]
Is it possible to write a wxGo program that reads a glade or xml file created by a form designer (similar to the way the GtkBuilder object works with GTK)?

**Comment by Kvaz1r** [comment]
No, there isn't support for something like this now. I remember similar issue in wxFormbuilder - https://github.com/wxFormBuilder/wxFormBuilder/issues/477 , maybe one day someone fix it.

**Comment by Asday** [comment]
wxFormBuilder's save file format is pretty sensible XML.  I got half way through writing a custom Python importer that would let one `import formbuildersavefile` a while back.

And I'm an _idiot_.  I'm sure you go programmers can smash something great out in short order.

**Comment by dontpanic92** [comment]
wxFormBuilder can generate XML format dialog definitions which can be loaded using `wx.XmlResourceGet().Load()`. For how to use `XmlResource` you can refer to wx [Doc](https://docs.wxwidgets.org/3.1/classwx_xml_resource.html)
